config/schedule.rb

```diff
@@ -16,8 +16,10 @@
   runner 'WorkerHealthChecker.enqueue_dummy_jobs'
 end
 
-mail_batch = Whenever.seconds(Figaro.env.usps_mail_batch_hours.to_i, :hours)
+if Figaro.env.enable_usps_verification == 'true'
```

Comment:
Looks like everywhere else we're doing this check we use `FeatureManagement.enable_usps_verification?`.